composer.json

```diff
@@ -32,8 +32,10 @@
 		"contributte/mailing": "^0.5.2",
 		"contributte/menu-control": "^2.2",
 		"contributte/monolog": "^0.5.0",
+		"contributte/psr6-caching": "^0.2.1",
 		"contributte/recaptcha": "^3.4",
 		"contributte/translation": "^1.0",
+		"fmasa/messenger": "dev-feature/handler-options as 2.0.0",
```

Comment:
Can we create the PR?

Reply:
Firstly, tests are needed

Reply:
PR created: https://github.com/fmasa/messenger/pull/17